Service/LoggerManagement.php

```diff
@@ -84,6 +84,7 @@ public function addEvent(
         $log->setSeverity($severity);
         $log->setIdentifierLabel($identifierLabel);
         $log->setIdentifierValue($identifierValue);
+        $log->setTimestamp(strtotime(time()));
```

Comment:
@RakeshJesadiya

![image](https://user-images.githubusercontent.com/1770746/207300713-179e0a35-92db-47bb-82a4-01325d3a3fcc.png)

How was that supposed to work?

@bartoszkubicki

Reply:
@pknap It's might not have worked but while I was testing on woodies it was working and added a time log to the database table.
I don't see any error while testing and still it is working but it's syntactically wrong so I will update the PR.

Reply:
@pknap Yes, It's (strtotime(time())) still working on PHP. that's the magic of PHP or whatever not sure. **But that must need to be thrown an error but it was working locally while I tested it that's not correct.**

**strtotime(time())** while run time needs to throw an error but that will bypass it in woodies local and add the current time in the DB field.